cmd/tempo-cli/cmd-list-blocks.go

```diff
@@ -16,8 +16,9 @@ import (
 )
 
 type listBlocksCmd struct {
-	TenantID  string `arg:"" help:"tenant-id within the bucket"`
-	LoadIndex bool   `help:"load block indexes and display additional information"`
+	TenantID         string `arg:"" help:"tenant-id within the bucket"`
+	LoadIndex        bool   `help:"load block indexes and display additional information"`
+	IncludeCompacted bool   `help:"include compacted blocks"`
```

Comment:
```suggestion
	TenantID         string `arg:"" Help:"tenant-id within the bucket."`
	LoadIndex        bool   `help: "Load block indexes and display additional information."`
	IncludeCompacted bool   `help:"Include compacted blocks."`
```

Reply:
Lines 19 and 20 are changing field tags which we do not want.   Generally speaking, this area is the help text part of the command line program and changes can be kind of tricky.   For now let's leave it as-is, and look at a separate initiative to review the user-facing text of the cli.